### DISCUSSION TITLE
strelka-fileshot: command not found

### DISCUSSION THREAD

**Discussion by yorkyman** [target/strelka#211] [Q&A] [answered]
ive installed everything with docker, ran `sudo docker-compose -f build/docker-compose.yaml up`

![image](https://user-images.githubusercontent.com/78002373/174696764-718db2ce-8ed1-4a62-b81a-6676368b56cb.png)

In the second terminal when i run `strelka-fileshot -c fileshot.yaml`
![image](https://user-images.githubusercontent.com/78002373/174696980-26efa027-daa0-4efe-87b9-bd126a1e1f4f.png)

i get a strelka-fileshot: command not found

Any help with this?

**Comment by phutelmyer** [comment, marked as answer]
Hi @yorkyman,

You may have to [first build](https://github.com/target/strelka#step-2-build-strelka-fileshot-file-submitter) `strelka-fileshot`.

Once the Strelka cluster is up (which is what you're seeing in the first image), you need to be in the `strelka` directory and build [strelka-fileshot](https://github.com/target/strelka#step-2-build-strelka-fileshot-file-submitter). After that, you can run `./strelka-fileshot -c fileshot.yaml` where [fileshot.yaml points to one or may folders that hold the files you want to send.](https://github.com/target/strelka/blob/master/configs/go/fileshot/fileshot.yaml)

I hope to release a Web UI that allows you to submit files to a Strelka cluster without having to build `strelka-fileshot` or using the CLI. I plan on releasing it in the next few weeks.

**Reply by yorkyman** [reply]
Any solution for 
![image](https://user-images.githubusercontent.com/78002373/174927572-9a5d5511-7c26-45bc-a0c0-f789a76bd105.png)

**Reply by phutelmyer** [reply]
@yorkyman Haven't seen that error before, but searching around...seems to be a `git` issue? [The version of linux you may be using may not be compatible with building](https://github.com/golang/go/issues/51253).

You might be able to try:
`-buildvcs=false`
when running go build to see if it passes.

You can also try setting this and trying again:
`git -c log.showsignature=false show`

Finally, if you put a -v on your build, you might get more details.
